secant/UIComponents/FontStyles/SecantTextStyles.swift

```diff
@@ -21,6 +21,10 @@ extension Text {
         self.modifier(TitleTextStyle())
     }
 
+    func synopsisText() -> some View {
```

Comment:
Should the function name be changed to represent the style being used since this is global to any view?

Reply:
updated to 
```swift
.paragraphText()
```